R/finalize_plot.R

```diff
@@ -214,26 +219,38 @@ finalize_plot <- function(plot = NULL,
     list(
       height = grid::convertUnit(unit(height, "in"), "bigpts", valueOnly = TRUE),
       width = grid::convertUnit(unit(width, "in"), "bigpts", valueOnly = TRUE),
-      title_width = grid::convertUnit(unit(title_width, "in"), "bigpts", valueOnly = TRUE),
-      margin_title_to_top = consts$margin_topline_t + consts$margin_title_t
```

Comment:
Note that I removed this helper margin combo because it was now only accurate for the horizontal (and not the vertical) layout.

Reply:
This makes lots of sense to me. I agree that it was starting to feel superfluous.